AppSettingsManagerBff/AppSettingsManagerBff.Model/Requests/UpdateUserPasswordRequest.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace AppSettingsManagerBff.Model.Requests;
+
+public class UpdateUserPasswordRequest
+{
+    public string UserId { get; set; }
+    public string OldPassword { get; set; }
+    public string NewPassword { get; set; }
+}
```

Comment:
All the Permissions files are looking good! Since I did Permissions for the API side of things, your part will help us to get this  working ASAP.